packages/api/src/promise/Api.spec.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017-2019 @polkadot/api authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import ApiPromise from '../promise';
+import { ApiOptions } from '../types';
+import Hash from '../../../types/src/type/Hash';
```

Comment:
This is on the border of a nitpick since it is a test, however - would rather import this from ‘@polkadot/types’ (Same 2 lines below) instead of relative.

Reply:
done